src/context/wishlist-context.js

```diff
@@ -0,0 +1,70 @@
+import {createContext, useContext, useReducer} from "react";
+import { wishlistReducer } from "../reducer";
+import axios from "axios";
+
+const WishlistContext = createContext(null);
+
+const encodedToken = localStorage.getItem("encodedToken");
```

Comment:
Here also you can store the token in a context and access it from there.

Reply:
Okay, thanks for the suggestion.